add value to radio button for public collections

carylwyatt opened the pull request:
Problem: when editing a collection via the collection list "edit" modal, changing the collection from private to public doesn't actually work.

Fix: The only line that's actually different is 114, where I added `value="1"`. Previously, there was no value for that radio switch, which was causing the problem.

moseshll reviewed (approved):
This looks good. The linter changes also make it more readable compared to the "before" diff.

carylwyatt commented:
Updating the value on the input wasn't enough. The "shrd" query param was not getting updated, even with `value="1"` in the input. I updated the svelte binding on the radio inputs _and_ the value. 

I was having a hard time conceptualizing what was going wrong, so I dropped it in the svelte REPL. Take a look at the before and after (if you want): https://svelte.dev/repl/ec5b90df96ec4f7c897991797f10f85d?version=4.2.1

### test it
_These instructions assume you have at least one collection saved in collection builder._
- Head to [collection builder on dev-3](https://dev-3.babel.hathitrust.org/cgi/mb?a=listcs;colltype=my-collections)
- Log in
- Go to "my collections" (from the left sidebar)
- Choose a collection to edit, click the title to go to the collection page
- In the collection toolbar at the top of the main section, select the "Edit" button
- Toggle your private/public option and update
- If your private collection is now public (or vice versa), it worked!

moseshll reviewed (approved):
This is good. Editing succeeds.
